### PR TITLE
RIGA-511: Add a feature for Emergency Notifications

### PR DESCRIPTION
**Pull request by montgomeryyoung**
## Summary / Approach
This PR adds a feature which contains an Emergency Notification content type.

## Metadata

| Question | Answer |
|----------|--------|
| Did you apply meaningful labels to the pull request? | yes
| Documentation reflects changes? | yes
| `CHANGELOG` reflects changes? | yes
| Unit/Functional tests cover changes? | n/a
| Did you perform browser testing? | yes
| Did you provide detail in the summary on how and where to test this branch? | no
| Risk level | low
| Relevant links | [RIGA-511](https://thinkoomph.jira.com/browse/RIGA-511)

### CODE REVIEW

File: ecms_base/features/custom/ecms_emergency_notification/ecms_emergency_notification.info.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  - 'publishcontent:publishcontent'
+  - 'rabbit_hole:rabbit_hole'
+  - 'redirect:redirect'
+  - 'role_delegation:role_delegation'
+  - 'scheduled_transitions:scheduled_transitions'
+  - 'simple_menu_permissions:simple_menu_permissions'
+  - 'simple_sitemap:simple_sitemap'
+  - 'webform:webform'
+  - 'webform:webform_node'
+  - 'webform_encrypt:webform_encrypt'
```
Comment (pfrilling) [inline ecms_base/features/custom/ecms_emergency_notification/ecms_emergency_notification.info.yml:56]: Why is webform listed as a dependency? There are many others that don't seem to fit here?

File: ecms_base/features/custom/ecms_emergency_notification/ecms_emergency_notification.info.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+name: 'Emergency Notification'
+description: 'Provides Emergency Notifications'
+type: module
+core_version_requirement: '^9.4 || ^10'
```
Comment (pfrilling) [inline ecms_base/features/custom/ecms_emergency_notification/ecms_emergency_notification.info.yml:4]: Add 11 too.

**Review by pfrilling — COMMENTED** [review]
A few questionable dependencies are listed. Not sure why.

File: ecms_base/features/custom/ecms_emergency_notification/ecms_emergency_notification.info.yml (hunk 1 above)
Comment (montgomeryyoung) [reply ecms_base/features/custom/ecms_emergency_notification/ecms_emergency_notification.info.yml:56]: The dependencies are set by the features module. I am hesitant to remove dependencies when I don't know why they are there.

File: ecms_base/ecms_base.install [hunk 3]
```diff
@@ -2206,3 +2206,14 @@ function ecms_base_update_10203(array &$sandbox): void {
     $active_storage->write("{$configBlock}", $update_source->read("{$configBlock}"));
   }
 }
+
+/**
+ * Install the ecms_emergency_notification feature.
+ */
+function ecms_base_update_10204(array &$sandbox): void {
```
Comment (pfrilling) [inline ecms_base/ecms_base.install:2213]: This is updating existing sites, but how is this feature being installed on new sites?

Comment (montgomeryyoung) [reply ecms_base/ecms_base.install:2213]: I added it to the ecms_base.info

File: ecms_base/features/custom/ecms_emergency_notification/ecms_emergency_notification.info.yml (hunk 1 above)
Comment (montgomeryyoung) [reply ecms_base/features/custom/ecms_emergency_notification/ecms_emergency_notification.info.yml]: I removed excess dependencies

**Review by pfrilling — APPROVED** [review]
Looks good

**Pull request merged**